unit-tests/signaturetests.cpp

```diff
@@ -270,13 +306,13 @@ bool signature_single_type2() {
         } \
     } while( 0 )
 
-int main( int argc, char** argv ) {
-    if( argc < 1 ) {
+int main( int argc, char** argv )
+{
+	if ( argc < 2 )
         return 1;
-    }
 
-    std::string test_name = argv[1];
-    bool ret = false;
+    std::string test_name(argv[1]);
+    bool ret = true;
```

Comment:
this needs to be initialized as `false`, because if a test is not found it needs to fail.